_guides/index.md

```diff
@@ -26,5 +27,47 @@ than what is included, limited only by what you want to achieve. The Plugin API
 is also intended as a way to simplify testing by providing users a way to
 encapsulate common assertions for repeat use.
 
+### Exposing Globals in Plugins
+
+When creating a Chai plugin, it's possible to expose globals that can be used across multiple files. Here's how to do it sustainably:
+
+#### Good Practice
+
+Expose any global returned in the `chai.use()` in the module record as well, so it can be imported directly:
+
+```javascript
+// An example of a good plugin:
+
+export const myGlobal = {...};
+
+export default function myPlugin(chai, utils) {
+  chai.myGlobal = myGlobal;
```

Comment:
i wonder if we should start advising against mutating the chai root

do you know of any existing plugins which couldn't just export their own functions instead of mutating chai?

(note i only mean mutating the root object, not adding extension methods)

Reply:
Yeah I think you're right. I'll change this.